### DISCUSSION TITLE
[Bug] Saved replies are only half readable in Firefox

### DISCUSSION THREAD

**Discussion by SMillerDev** [community/community#4292] [Pull Requests]
Saved replies in Pull request responses are cut off in Firefox.

![Screenshot 2021-06-15 at 11 33 13](https://user-images.githubusercontent.com/1484494/123240180-60661e00-d4e0-11eb-9920-c8f894c31877.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
